pandas/core/panel.py

```diff
@@ -197,7 +199,7 @@ def _init_dict(self, data, axes, dtype=None):
             haxis = Index(ks)
 
         for k, v in compat.iteritems(data):
-            if isinstance(v, dict):
+            if isinstance(v, MutableMapping):
```

Comment:
I think ``is_dict_like`` should *also* test for ``MutableMapping`` (and we shouldn't test for this outside of ``is_dict_like``).